ui/src/components/dynamics-form/constructor/items/MultiSelectConstructor.vue

```diff
@@ -111,12 +117,14 @@ const getData = () => {
 const rander = (form_data: any) => {
   formValue.value.option_list = form_data.option_list || []
   formValue.value.default_value = form_data.default_value
+  formValue.value.show_default_value = form_data.show_default_value
 }
 
 defineExpose({ getData, rander })
 onMounted(() => {
   formValue.value.option_list = []
   formValue.value.default_value = ''
+  formValue.value.show_default_value = true
 
   addOption()
 })
```

Comment:
Your `rander` function currently takes over setting the initial values of form properties and adding options to the list. This can be streamlined slightly without changing functionality.

Here are some suggestions:

1. **Consolidate Option List Reset**: You can combine the resetting of `option_list`, `default_value`, and `show_default_value` into single lines in the `onMounted` hook.
2. **Optimize Data Initialization**: Initialize `default_value` directly within `rander` instead of passing an empty string when it's not necessary to change the default value.

Here is an optimized version of your code:

```javascript
const getData = () => {
  formData.option_list = formData.option_list || [];
}

// Simplify onMounted setup
onMounted(() => {
  formValue.value.option_list = [];
  // Avoid initializing show_default_value here or using formData.show_default_value because it is handled elsewhere
})

defineExpose({
  getData,
  rander
})
```

This way, you reduce unnecessary reassignments and streamline the initialization process.